java/src/jmri/jmrit/consisttool/ConsistTool.properties

```diff
@@ -53,3 +53,4 @@ ScanConsists = Scan Roster for Consists
 WriteCVToRosterOption = Update CV19 in roster when modifying advanced consists
 WriteCVToRosterToolTip = When modifying advanced consists, CV19 is changed.  Selecting this option to update the roster entry, when known, when CV19 is updated.
 StartupConsistToolAction=Open Consisting Tool
+ConsistPreferencesToolTip = Preferences for the consist tool.
```

Comment:
Tooltips should only be used when they can provide *more* information, and a tooltip displayed in the Preferences window over "Consist Control" that reads "Preferences for the consist tool" doesn't. I would remove this.

Reply:
thanks, will fix